### PR TITLE
[IMP] contract: Add contract lines entry point for reporting

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

File: contract/views/contract_line.xml [hunk 1]
```diff
@@ -193,4 +193,108 @@
...
+        <field name="domain">[('contract_id.contract_type', '=', 'purchase')]</field>
+        <field name="context">
+            {'search_default_group_by_contract': 1}
+        </field>
+        <field name="search_view_id" ref="contract_line_search_view" />
+        <field name="view_id" ref="contract_line_report_tree_view" />
+    </record>
+
+    <record id="action_customer_contract_lines" model="ir.actions.act_window">
+        <field name="name">Customer Contract Lines</field>
```
Comment (pedrobaeza) [inline contract/views/contract_line.xml:276]: Why 2 separate actions instead of one like invoice analysis?

**Review by angelmoya — APPROVED** [review]

File: contract/views/contract_line.xml (hunk 1 above)
Comment (rousseldenis) [reply contract/views/contract_line.xml:276]: As there are two menus for supplier and customer contracts.

Some maybe want to give access to one or other.

Comment (pedrobaeza) [reply contract/views/contract_line.xml:276]: OK

**Pull request merged**